ATFConverter/Tokenizer.py

```diff
@@ -7,64 +7,90 @@
 class Tokenizer(object):
 
     def line_tokenizer(self, text):
-    
+        """
+        Takes .txt file, reads it, separates lines, and prints only line numbered lines as list of strings (ignores metadata).
+        Text must be a string.
+        """
         line_output = []
 
-        with open(text, mode = "r+", encoding = "utf8") as File:
+        with open(text, mode='r+', encoding='utf8') as File:
             lines = File.readlines()
-            for no_meta in lines: 
-                if re.match(r'^\d*\.|\d\'\.', no_meta):
-                    try:
-                        line_output.append(no_meta) 
-                        #no_meta = re.subn(r'\n', '', no_meta) # doesn't work.
-                    except KeyError:
-                        line_output.append("ERROR")
+            assert isinstance(text, str), 'Incoming argument must be a string.'
+            for no_metadata in lines:
```

Comment:
Just a small point, but what you iterating over here is "lines" so it should just be:
`for line in lines:`
the qualifier of no_metadata hasn't been made yet, so it's not a descriptive variable name.